Doc/tools/extensions/pyspecific.py

```diff
@@ -430,6 +430,7 @@ def run(self):
 # Support for including Misc/NEWS
 
 issue_re = re.compile('(?:[Ii]ssue #|bpo-)([0-9]+)')
+gh_issue_re = re.compile('(?:gh-issue-)([0-9]+)')
```

Comment:
@ezio-melotti 
Are you going to re-generate all the NEWS.d after https://github.com/python/core-workflow/pull/451 is landed?

Reply:
or should I update it to support `gh-issue-xxx` and `gh-xxxx`?